src/View/Admin/WooTab.php

```diff
@@ -555,39 +675,19 @@ public function maybe_add_connection_button( $settings ) {
 	}
 
 	/**
-	 * Add the Constant Contact connection button when displaying the form.
-	 *
-	 * Will display as a "Disconnect" button if the connection has already been established.
-	 *
-	 * @since  2019-03-08
-	 * @author Zach Owen <zach@webdevstudios>
-	 */
-	public function add_cc_connection_button() {
-		$connected = get_option( ConnectionStatus::CC_CONNECTION_ESTABLISHED_KEY );
-		$value     = $connected ? 'disconnect' : 'connect';
-		$message   = $connected
-			? esc_html__( 'Disconnect from Constant Contact', 'cc-woo' )
-			: esc_html__( 'Connect with Constant Contact', 'cc-woo' );
-
-		wp_nonce_field( $this->nonce_action, $this->nonce_name );
-		?>
-		<div style="padding: 1rem 0;">
-		<button class="button button-primary" type="submit" name="cc_woo_action" value="<?php echo esc_attr( $value ); ?>">
-			<?php echo esc_html( $message ); ?>
-		</button>
-			<span style="line-height:28px; margin-left:25px;">
-				<?php
-				printf(
-					/* translators: the placeholders hold opening and closing `<a>` tags. */
-					esc_html__( 'If you have any issues connecting please call %1$sConstant Contact Support%2$s', 'cc-woo' ),
-					'<a href="https://community.constantcontact.com/contact-support">',
-					'</a>'
-				);
-				?>
-			</span>
-		</div>
-		<?php
-	}
+	* Add a go back button.
+	*
+	* @since  2022-06-23
```

Comment:
For the time being, this will be the last I point out date usage, as you know already.

We'll get these handled before final release :D